devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy

```diff
@@ -399,6 +402,14 @@ def configure() {
 
 	setConfigured("true")
 
+	// set the check interval based on the report interval preference. (default 122 minutes)
+	// we do this here in case the device is in wakeup mode
+	def checkInterval = 2 * 60 * 60 + 2 * 60
+	if (reportInterval) {
+		checkInterval = 2*(timeOptionValueMap[reportInterval]/60) + (2 * 60)
```

Comment:
The value for `"18 hours"` in `getTimeOptionValueMap` is actually only 6 hours

Reply:
That's a good catch. Looks like it's wrong in the aeon multisensor (original) code too. I'll go ahead and fix both under this commit.

Reply:
Math is wrong here. You're dividing the seconds by 60 but checkInterval is in seconds too.